app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
To remove this comments add this to the **rubocop.yml**:
```
Style/FrozenStringLiteralComment:
  Enabled: false
```

Reply:
Sounds great!